### ISSUE TITLE
Use the term "proposal" for SCM changes?

### ISSUE THREAD

**Issue by e-backmark-ericsson** [cdevents/spec#62]
There was an old [PR in the old sig-events repo](https://github.com/cdfoundation/sig-events/pull/85) on introducing the term "proposal" for some of our SCM event(s). Much has happened in the spec since then and the spec is moved to cdevents, so a new PR needs to be created based on the current codebase. I've closed that old PR in favor of this issue.

Is that proposal still relevant @erkist ?

**Comment by afrittoli** [comment]
Related to https://github.com/cdevents/spec/issues/65
See the SIG Interoperability doc: https://github.com/cdfoundation/sig-interoperability/blob/main/docs/tools-terminology.md#scm-tools-and-technologies

We should do the change only once, when aligning to the SIG interoperability terminology.

**Comment by afrittoli** [comment]
As discussed during CDEvents WG on 2.9.2024 - there doesn't seem to be a strong enough motivation to justify the change at this point, so the proposal is to close the issue.
@xibz would you agree?
